### ISSUE TITLE
Can the controls for the classes be renamed for the next release?

### ISSUE THREAD

**Issue by SimonRPSM** [ryanpopo/Linux-RT-Libraries#1]
Currently, there are multiple controls that are called the same as members of different classes (ex, Mutex  and Mutex Attribute both have a "reference" control). 

Could they be renamed such that they do not have the exact same name? This would be handy for building stuff into LLBs or other build processes that enforce a flat structure. I tried it out on my machine and there were no issues with doing this, and it does not change any external usage of the APIs (but would require updating typedefs that link to that typedef).

**Comment by ryanpopo** [comment]
Yeah I agree this would make things way simpler for flattening the VIs down for an LLB. There shouldn't be any reason for not doing this.

**Comment by SimonRPSM** [comment]
I can make this change then.
